backend/app/testbot/llm.py

```
@@ -0,0 +1,32 @@
+# llm.py
+import json
+import requests
+from config import GEMINI_API_URL
+
+def ask_llm(prompt):
+    """
+    Given a prompt, call the Gemini API and return the generated answer.
+    """
+    payload = {
+        "contents": [
+            {"parts": [{"text": prompt}]}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)